tensorflow/lite/micro/tools/make/flatbuffers_download.sh

```diff
@@ -32,6 +32,11 @@
 set -e
 
 SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# If the TENSORFLOW_ROOT command line flag is mentioned during the build,
+# like $TENSORFLOW_ROOT=tflite-micro/, then we can simply jump to that
+# particular directory and run the process. If that is not mentioned, then
+# we following the old path of moving up from the SCRIPT_DIR.
```

Comment:
thinking some more about this, let's change the parameter to the script to be the path to the downloads folder relative to the location from where the script is called. Then we can remove the ROOT_DIR={SCRIPT_DIR}/../../../ etc. and have the script be much simpler.

Reply:
As discussed, we are passing the download directory relative to the `pwd` and passing the second parameter TENSORFLOW_ROOT to use the bash_helper scripts.

Reply:
Changed according to our discussion.